homeassistant/components/plex/services.py

```diff
@@ -77,17 +76,22 @@ def refresh_library(hass: HomeAssistant, service_call: ServiceCall) -> None:
     library.update()
 
 
-def get_plex_server(hass, plex_server_name=None, plex_server_id=None):
+def get_plex_server(
+    hass: HomeAssistant,
+    plex_server_name: str | None = None,
+    plex_server_id: str | None = None,
+) -> PlexServer:
     """Retrieve a configured Plex server by name."""
     if DOMAIN not in hass.data:
         raise HomeAssistantError("Plex integration not configured")
-    plex_servers = hass.data[DOMAIN][SERVERS].values()
-    if not plex_servers:
+    servers: dict[str, PlexServer] = hass.data[DOMAIN][SERVERS]
```

Comment:
Small change here to first get the dict, and only get the `values()` on line 94.